arwen/elrondapi/bigIntOps.go

```diff
@@ -633,9 +633,6 @@ func v1_4_bigIntSetSignedBytes(context unsafe.Pointer, destinationHandle int32,
 
 	value := managedType.GetBigIntOrCreate(destinationHandle)
 	twos.SetBytes(value, bytes)
-
```

Comment:
You have to delete the same thing above as well. on bigIntSetUnsignedBytes.

Reply:
done